__tests__/.eslintrc

```diff
@@ -1,20 +1,3 @@
 {
-  "extends": ["@readme/eslint-config/testing"],
-  "env": {
-    "jest": true
-  },
-  "rules": {
-    "jest/expect-expect": [
```

Comment:
All tests have explicit `expect()`s now so we don't need add these exclusions for nock.